Strings/NPCs/NPCs.md

```diff
@@ -1,28 +1,82 @@
-﻿# Lone Shadow Masamume the Crimson-Fang
+# Red Guard Elite - Yamagata Masashige
 
 | Language | Name |
 | :------- | :---------- |
-| Japanese | 孤影衆　紅牙の正宗 |
+| Text ID | 911042 |
+|||
+| Japanese | 赤備え武者　山縣昌茂 |
 | Korean | Translation needed |
-| German | Einsamer Schatten Masamume die blutrote Klinge |
-| Italian | Ombra Solitaria Masamune la Zanna Cremisi |
-| Spanish | Sombra Solitaria Masamune lo Colmillo Carmesí |
+| German | Translation needed |
+| Italian | Translation needed |
+| Spanish | Translation needed |
 
 # Kamikaze Enthusiast - Kaiten Shi
 
 | Language | Name |
 | :------- | :---------- |
-| Japanese | Translation needed |
+| Text ID | 911054 |
+|||
+| Japanese | 寄鷹衆　神風のカイテン <br> Nightjar - Kaiten of Kamikaze|
 | Korean | Translation needed |
 | German | Kamikaze-Enthusiast Kaiten Shi |
 | Italian | Translation needed |
 | Spanish | Translation needed |
 
+# Samurai General Ramiresu Arumando (Unused)
```

Comment:
If its unused, might not make sense to add it unless its being put in game for sure. 
If so please let me know.

Reply:
> 
> 
> he is here because most of work here was also for my translation patch, which can be used for the older version of S:R (before Ramiresu removed)

@PrattiDev Can you comment if this can be removed in this patch if not needed? Maybe the translations can be kept as a record?